### PR TITLE
chore: remove dashboard feature switch

### PR DESCRIPTION
**Pull request by johnkim-det**
## Description

https://determinedai.atlassian.net/browse/WEB-860

## Test Plan

Dashboard should appear as an option in the navigation bar, and should be the default route (unless rbac is on, in which case the default route is the workspaces list page).

## Commentary (optional)

Haven't received design approval on the changes from WEB-814, not sure if we should wait on that to merge this PR to release the dashboard.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Review by ClaireNeveu — APPROVED** [review]

**Comment by ClaireNeveu** [comment]
The design changes look good but I notice the JupyterLab button is disabled on the netlify preview. Shouldn't that be enabled since rbac is off?

**Comment by ClaireNeveu** [comment]
Also noticed that the error message is wrong and using jargon. I thought we had corrected that to say "You do not have permission to launch JupyterLab".

**Comment by julian-determined-ai** [comment]
@johnkim-det related to the `disabled` button that Claire mention can you update this line: 

https://github.com/determined-ai/determined/blob/c89234ccb7d7f89bf3d9ff0d6ec2d9ae7efe73bf/webui/react/src/pages/Dashboard.tsx#L182

to:

```
const JupyterLabButton = () => {
    return canCreateNSC ? (
      <Button onClick={() => openJupyterLabModal()}>Launch JupyterLab</Button>
    ) : (
      <Tooltip placement="leftBottom" title="User lacks permission to create NSC">
        <div>
          <Button disabled>Launch JupyterLab</Button>
        </div>
      </Tooltip>
    );
  };
```

File: webui/react/src/components/TaskList.tsx [hunk 1]
```diff
@@ -116,8 +115,6 @@ const TaskList: React.FC<Props> = ({ workspace }: Props) => {
   const [tasks, setTasks] = useState<CommandTask[] | undefined>(undefined);
   const [sourcesModal, setSourcesModal] = useState<SourceInfo>();
   const pageRef = useRef<HTMLElement>(null);
-  const { contextHolder: modalJupyterLabContextHolder, modalOpen: openJupyterLabModal } =
```
Comment (julian-determined-ai) [inline webui/react/src/components/TaskList.tsx:119]: When a user is on the Task list within a workspace the dropdown should be required and default to the current workspace. It looks like due to removing the `workspace` prop in `useModalJupyterLab` this is no longer working correctly. 

2A2D6B35-624C-4169-982A-4F3B6D2E65F1

**Review by julian-determined-ai — CHANGES_REQUESTED** [review]

File: webui/react/src/components/TaskList.tsx (hunk 1 above)
Comment (johnkim-det) [reply webui/react/src/components/TaskList.tsx:119]: ah thanks for catching that, updated

**Review by julian-determined-ai — APPROVED** [review]
LGTM!

**Pull request merged**